mesa/model.py

```diff
@@ -77,6 +81,29 @@ def read_csv_city(self, filename: str):
         """
         return pd.read_csv(os.path.join(self.data_folder, filename + ".csv"))
 
+    def change_global_gender_ratio(self, target_ratio):
```

Comment:
In questa procedura c'è una buona porzione di codice ripetuto.

Reply:
Io questa procedura non la cambierei. E' scritta così in Netlogo dalle prime versioni. Sono due cicli che si differenziano nel modo di scaricare i risultati, concatenate.

Reply:
Ottimo! 😄